appserver/extras/payara-micro/payara-micro-boot/src/main/resources/MICRO-INF/domain/branding/glassfish-version.properties

```diff
@@ -3,7 +3,7 @@ brief_product_name=@@@PRODUCT@@@
 abbrev_product_name=@@@PRODUCT@@@
 major_version=5
 minor_version=2021
-update_version=7
+update_version=8
```

Comment:
```suggestion
update_version=8-SNAPSHOT
```

Reply:
The scripts aren't designed for snapshots in the glassfish-version.properties file, it's been working like this for as long as I can remember